athf/agents/llm/hunt_researcher.py

```
@@ -582,11 +616,12 @@ def _llm_summarize_system_research(
                 ' "finding3"]\n}}'
             ).format(topic=topic, context=context)
 
-            response = self._call_llm(prompt, max_tokens=2048)
+            response = self._call_llm(prompt, max_tokens=2048, response_format="json")
             data = self._parse_json_response(response)
             return data["summary"], data["key_findings"]
 
         except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)